modules/nextgenmap/main.nf

```diff
@@ -0,0 +1,58 @@
+process NEXTGENMAP {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
I would expect process_high for an aligner, as they generally scale linearly with number of cpus.

Reply:
I wish there was a `high_cpu_low_mem` label. The amount of memory that `process_high` requests is overkill for this program.

`npm` is quite fast. The time issue that I run into most on my cluster with this program is not the time it takes to align, but the time it takes to wait in the queue for resources. But, I defer to other opinions on this. Admittedly, I work mostly on smaller genomes.

Reply:
I see that bbmap is also on process_medium, so there is precedent.

Reply:
I'm going to leave it at process_medium and get it merged.